.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - 0.6
-  - 0.8
-  - 0.9
-branches:
-  only:
-    - master
-    - develop
\ No newline at end of file
+  - 0.10
+  - 4
+  - 6
+before_install:
+  "npm install npm@latest -g"
```

Comment:
This was a hack for older node versions. I'd say drop support for anything below 4 and be done with it.